src/main/java/dk/kb/storage/storage/DsStorage.java

```diff
@@ -44,13 +44,17 @@ public class DsStorage implements AutoCloseable {
     private static final String CTIME_COLUMN = "ctime";
     private static final String MTIME_COLUMN = "mtime";
     private static final String PARENT_ID_COLUMN = "parentid";
+    private static final String KALTURA_REFERENCE_ID_COLUMN = "kalturareferenceid";
+    private static final String KALTURA_INTERNAL_ID_COLUMN = "kalturainternalid";
```

Comment:
I might need an explanation here. What is the difference between referenceId and internalId? And if they are not the same, what is the difference between the `id` of the record and its `kalturaReferenceId`

Reply:
ReferenceId is the value we have given to it. The value in kaltura is "referenceId".